File: src/me/memorial/module/modules/world/FastPlace.java
package me.memorial.module.modules.world;

import me.memorial.module.Module;
import me.memorial.module.ModuleCategory;
import me.memorial.module.ModuleInfo;
import me.memorial.value.IntegerValue;

@ModuleInfo(name = "FastPlace", description = "Allows you to place blocks faster.", category = ModuleCategory.WORLD)
public class FastPlace extends Module {

    public IntegerValue speedValue = new IntegerValue("Speed", 0, 0, 4);

    public static FastPlace instance;

    public FastPlace() {
        instance = this;
    }
}

File: src/me/memorial/module/modules/world/FastPlace.kt
package me.memorial.module.modules.world

import me.memorial.module.Module
import me.memorial.module.ModuleCategory
import me.memorial.module.ModuleInfo
import me.memorial.value.IntegerValue

@ModuleInfo(name = "FastPlace", description = "Allows you to place blocks faster.", category = ModuleCategory.WORLD)
class FastPlace : Module() {
    val speedValue = IntegerValue("Speed", 0, 0, 4)

    companion object {
        var instance: FastPlace? = null
    }

    init {
        instance = this
    }
}
